### DISCUSSION TITLE
GE meeting Monday March 20 18:30 UTC

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#112] [Announcements]
Hope to see some of the @delph-in/participants this coming Monday!

Zoom: 926 4912 4250
Password: 674288

https://cl-cam-ac-uk.zoom.us/j/92649124250?pwd=L2YzUE1pU3NHSmlsVDVnS203VmpNdz09

Hosted kindly by @wsun106

**Comment by olzama** [comment]
A reminder that this is today!

**Comment by fcbond** [comment]
I plan to be there

On Mon, 20 Mar 2023 at 12:53, Olga Zamaraeva ***@***.***>
wrote:

> A reminder that this is today!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/112#discussioncomment-5368526>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRQZH2EJL4CRG7VORWLW5BAKDANCNFSM6AAAAAAV5CKUSY>
> .
> You are receiving this because you are on a team that was mentioned.Message
> ID: ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by emilymbender** [comment]
In transit so can't this time...

**Comment by arademaker** [comment]
I have another meeting conflicting, I will try to join

**Comment by danflick** [comment]
I'm connected, but get a message "the meeting has not started".

 Dan

**Comment by fcbond** [comment]
Me too

**Comment by arademaker** [comment]
<img width="566" alt="image" src="https://user-images.githubusercontent.com/86403/226434754-cb4da278-0dc2-46c5-a088-11ab0774c4ae.png">

**Comment by olzama** [comment]
Me too :(

Anyone else has proper zoom? @fcbond  maybe you could start an alternative meeting?

**Comment by olzama** [comment]
@wsun106 do we have you today? :)

**Comment by arademaker** [comment]
Can I offer my zoom ?

**Reply by olzama** [reply]
@arademaker yes please!

**Comment by arademaker** [comment]
https://fgv-br.zoom.us/my/arademaker

**Comment by fcbond** [comment]
I just made one, but let's use Alexandre's

On Mon, 20 Mar 2023 at 19:36, Alexandre Rademaker ***@***.***>
wrote:

> https://fgv-br.zoom.us/my/arademaker
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/discussions/112#discussioncomment-5373134>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRURVP3L3ADYD62OTOLW5CPSVANCNFSM6AAAAAAV5CKUSY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>
